modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/model/listener/APIEndpointRelevantObjectEntryModelListener.java

```diff
@@ -146,12 +149,25 @@ private void _validate(ObjectEntry objectEntry) {
 					_objectFieldLocalService.getObjectField(
 						objectEntry.getObjectDefinitionId(), "path");
 
+				String locale = objectField.getLabel(user.getLocale());
```

Comment:
It is not a locale, it is a label based on the locale, so the name is not apropriate, what about `String label` ? And I think BC will move just before it is being used